src/pact.integration.spec.ts

```diff
@@ -267,49 +257,35 @@ describe("Integration", () => {
 
       context("with an unexpected interaction", () => {
         // add interactions, as many as needed
-        before((done) => {
-          provider.addInteraction({
-            state: "i have a list of projects",
-            uponReceiving: "a request for projects",
-            withRequest: {
-              method: "GET",
-              path: "/projects",
-              headers: {
-                Accept: "application/json",
-              },
+        before(() => provider.addInteraction({
+          state: "i have a list of projects",
+          uponReceiving: "a request for projects",
+          withRequest: {
+            method: HTTPMethod.GET,
+            path: "/projects",
+            headers: {
+              Accept: "application/json",
             },
-            willRespondWith: {
-              status: 200,
-              headers: {
-                "Content-Type": "application/json",
-              },
-              body: EXPECTED_BODY,
+          },
+          willRespondWith: {
+            status: 200,
+            headers: {
+              "Content-Type": "application/json",
             },
-          }).then(() => done());
-        });
-
-        it("fails verification", (done) => {
-          const promiseResults: Array<Promise<any>> = [];
+            body: EXPECTED_BODY,
+          },
+        }).then(() => console.log("Adding interaction worked"), () => console.warn("Adding interaction failed.")));
 
+        it("fails verification", () => {
+          // TODO: fix this test, not sure what it's trying to do
```

Comment:
I think you've captured the intention. I assume it was about making two requests but only supplying one. I think we can remove the TODO.